### PR TITLE
Adds more info regarding `Infinity`

### PR DESCRIPTION
**Pull request by OnkarRuikar**
As per [the suggestion](https://github.com/mdn/content/pull/16596#issuecomment-1138713030) adds more content regarding  `Infinity`.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Do you think it's worth mentioning for `parseInt` as well? When I tried my hands on it I didn't find a way to cleverly make it in, so I gave up.

```js
parseInt("1" + "0".repeat(309)) === Infinity
```

File: files/en-us/web/javascript/reference/global_objects/parsefloat/index.md [hunk 1]
```diff
@@ -86,6 +87,15 @@ The following example returns `NaN`:
 parseFloat('FF2');
 ```
 
+### `parseFloat` returning `Infinity`
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/parsefloat/index.md:90]: No backticks in title, according our guidelines ([metadocs](https://github.com/mdn/content/blob/main/files/en-us/mdn/guidelines/writing_style_guide/index.md#headings))
```suggestion
### Returning infinity
```

File: files/en-us/web/javascript/reference/global_objects/parsefloat/index.md [hunk 2]
```diff
@@ -86,6 +87,15 @@ The following example returns `NaN`:
...
+
+Infinity values are returned when the number is outside the double-precision 64-bit IEEE 754-2019 format range:
+
+```js
+parseFloat('1.7976931348623159e+308');  //  Infinity
+parseFloat('-1.7976931348623159e+308'); // -Infinity
+```
+
 ### `parseFloat` and `BigInt`
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/parsefloat/index.md:99]:
```suggestion
### Interaction with BigInt values
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]

**Comment by OnkarRuikar** [comment]
@Josh-Cena To reduce the string size significantly we can use radix `36`. This is the closest I got:
```
parseInt('1A1E4VNGAIJNY80ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ' +
         'ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ' +
         'ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ', 36);
```

Or we can use some code to demonstrate:
```
const bigNumberString = BigInt(Number.MAX_VALUE) + BigInt(Number.MAX_VALUE) + '';
bigNumberString.length--;
parseInt(bigNumberString); // Infinity
```

**Comment by Josh-Cena** [comment]
Thanks for that info!

To be honest, I don't know if it's actually a special behavior of `parseInt`/`parseFloat`, or is it simply because there's no other way to represent it. The corresponding literals would have overflowed as well.

**Pull request merged**